assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -95,6 +95,9 @@ export default Component.extend({
       passive: true,
     });
     window.addEventListener("resize", this.onResizeHandler);
+    window.addEventListener("mousewheel", this.onScrollHandler, {
```

Comment:
This is due to ensure we are never stuck at the top without firing any scroll event, due to throttling and network it is possible to end up in this state.